app/server/src/main/scala/org/bitcoins/server/ChainRoutes.scala

```diff
@@ -35,6 +38,15 @@ case class ChainRoutes(chain: ChainApi)(implicit system: ActorSystem)
           Server.httpSuccess(hash)
         }
       }
+
+    case ServerCommand("getinfo", _) =>
```

Comment:
@benthecarman does this have a test someplace?

Reply:
No, can add one to `RoutesSpec`